### DISCUSSION TITLE
request.json

### DISCUSSION THREAD

**Discussion by drbaileyii** [strapi/strapi#6118] [Legacy Topics]
Was interested in using httponly cookies to secure the jwt.  I see the following in the request.json
under config/environment/production/request.json

```json
"session": {
    "enabled": true,
    "client": "cookie",
    "key": "strapi.sid",
    "prefix": "strapi:sess:",
    "secretKeys": ["mySecretKey1", "mySecretKey2"],
    "httpOnly": true,
    "maxAge": 86400000,
    "overwrite": true,
    "signed": false,
    "rolling": false
```

Does this mean it is implemented or is this future.. Confused on how this is used..

**Comment by lauriejim** [comment]
Hello! I don't understand what you are trying to do.
Can you please precise.
